azure/init.sh

```diff
@@ -6,13 +6,13 @@ set -u
 set -e
 # fail in a pipeline if any of the commands fails
 set -o pipefail
+# echo commands
+set -x
 
 # in redhat we need to enable default port for postgres
-firewall-cmd --add-port=5432/tcp
-
-# install azure client rpm to have correct certificates
```

Comment:
https://github.com/citusdata/test-automation/pull/173 

We don't need this weird thing anymore.